docs/mkdocs.yml

```diff
@@ -45,6 +45,7 @@ pages:
   - Tensorflow Support: ProgrammingGuide/tensorflow-support.md
   - Caffe Support: ProgrammingGuide/caffe-support.md
   - Quantization Support: ProgrammingGuide/quantization-support.md
+  - Create Layer/Criterion: ProgrammingGuide/add-layer-criterion.md
```

Comment:
May be we should change the title of this section to "Define new Layer/Criterion" or "Customizated Layer/Criterion", and similar. "Create layer/criterion" is a little bit confusing as it may means to create a layer or criterion object.